src/wallet/test/wallet_tests.cpp

```diff
@@ -586,6 +586,11 @@ BOOST_FIXTURE_TEST_CASE(wallet_disableprivkeys, TestChain100Setup)
     auto chain = interfaces::MakeChain(node);
     std::shared_ptr<CWallet> wallet = std::make_shared<CWallet>(chain.get(), WalletLocation(), WalletDatabase::CreateDummy());
     wallet->SetupLegacyScriptPubKeyMan();
+    {
+        auto locked_chain = chain->lock();
+        LOCK(wallet->cs_wallet);
+        wallet->SetLastBlockProcessed(::ChainActive().Height(), ::ChainActive().Tip()->GetBlockHash());
+    }
```

Comment:
Why is this needed? 7c73c4888f

Reply:
re: https://github.com/bitcoin/bitcoin/pull/16426#discussion_r417374233

> Why is this needed? [7c73c48](https://github.com/bitcoin/bitcoin/commit/7c73c4888f5c4c3f24a180f2841429318463db7d)

It does seem not to be needed anymore. Test passes for me with it commented out.